bin/configurer/utils/generate-local-user-mappings.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -eu
+
+swanseauserids=$(docker run -e PGPASSWORD='openidm' --rm --network ccd-network postgres:11-alpine psql --host shared-db  --username openidm --tuples-only  --command "SELECT string_agg(fullobject->>'_id',',') FROM managedObjects WHERE fullobject->>'userName' LIKE '%swansea%';" openidm)
```

Comment:
Can I ask you to use either snake case or camel case for the names? It is hard to read at the moment.

Reply:
yes of course :)